.github/workflows/docker.build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         with:
           commit-message: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
           title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
-          branch: deploy-to-staging-${{ steps.docker_meta.outputs.tags }}
+          branch: deploy-to-staging-${{GITHUB_RUN_ID}}
```

Comment:
I'm not entirely sure how to access env vars in action parameters. I can't find anything in the official docs right now, but random examples from Googling are all using `${{ env.GITHUB_RUN_ID }}`

Reply:
ok

Reply:
Ah, yeah looks like this should work https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-env-context